Guide

martindale opened the pull request:
This adds a new standalone file that contains a cleanly formatted guide, for export into outside tools.

heisler3030 commented:
Hey @martindale - you may want to update now that my updated README is merged.  I think your GUIDE.md can match it pretty closely.

martindale commented:
This is now updated, and split appropriately into `GUIDE.md` and `README.md`.  Once this is merged, I can import the content to its appropriate places.
